### PR TITLE
install bash script

### LINKED ISSUE #306: Review documentation

### LINKED ISSUE #319: Installation shell script
**Is your feature request related to a problem? Please describe.**
To simplify kubectl-kubtest plugin installation we can start with simple bash/sh script 

Part of #161

### PR DESCRIPTION
**Pull request by jasmingacic**
This script will detect your OS and fetch the latest version - 0.5.21 unless you override by seting env var `KUBTEST_VERSION`

for example 
```sh
$ KUBTEST_VERSION=0.5.19 ./install.sh
Getting kubectl-kubtest plugin
Downloading kubtest from URL: https://github.com/kubeshop/kubtest/releases/download/v0.5.19/kubtest_0.5.19_Linux_x86_64.tar.gz
kubectl-kubtest installed in /usr/local/bin/kubectl-kubtest
```

or by simply running 
```sh
$ ./install.sh
Getting kubectl-kubtest plugin
Downloading kubtest from URL: https://github.com/kubeshop/kubtest/releases/download/v0.5.21/kubtest_0.5.21_Linux_x86_64.tar.gz
kubectl-kubtest installed in /usr/local/bin/kubectl-kubtest
```

Ideally we will do something like 
```sh
$ curl -sSLf https://raw.githubusercontent.com/kubeshop/kubtest/install_bash_script/install.sh | sudo sh
```
the URL should be replaced by http://get.kubtest.io or similar

p.s. haven't tested it on windows

Signed-off-by: Jasmin Gacic <jasmin.gacic@gmail.com>

Fixes #306
Fixes #319

### CODE REVIEW

**Review by exu — DISMISSED** [review]
Please update docs with install command

**Review by nicufk — CHANGES_REQUESTED** [review]
LGTM but waiting for official repo get.kubtest.io.

**Review by hamzahsn — APPROVED** [review]
LGTM!

**Comment by jasmingacic** [comment]
Anyways with this PR install.sh will be moved to GH pages so the install command will be like:
```
$ curl -sSLf https://kubeshop.github.io/kubtest/install.sh | sudo bash
```

**Comment by jasmingacic** [comment]
I also added option to test GH pages locally with docker-compose. 

```
$ cd doc
$ docker-compose up
```

http://localhost:8080

**Review by nicufk — APPROVED** [review]
👍

File: install.sh [hunk 1]
```diff
@@ -1 +1,31 @@
 echo "Getting kubectl-kubtest plugin"
+#!/bin/sh 
+KUBTEST_VERSION=${KUBTEST_VERSION:-"0.5.21"}
```
[the remark this answers is not in the document]
Comment (jasmingacic) [reply install.sh:3]: so right now our latest release is  `v0.5.26-beta001` `https://github.com/kubeshop/kubtest/releases/latest`  sine v0.5.26 is less than v0.5.26-beta001 :D

**Pull request merged**